.travis.yml

```diff
@@ -3,9 +3,9 @@ language: c
 sudo: false
 
 env:
-  BYOND_MAJOR="510"
-  BYOND_MINOR="1346"
-  MACRO_COUNT=986
+  BYOND_MAJOR="511"
+  BYOND_MINOR="1381"
+  MACRO_COUNT=875
```

Comment:
What do you mean? I left a comment about it, and it has it's own commit. Wasn't trying to hide it or anything.